schemas/yaml/WordSet.yml

```diff
@@ -0,0 +1,87 @@
+$schema: 'http://json-schema.org/draft-07/schema#'
+$id:     'http://schemas.digitallinguistics.io/WordSet-1.0.0.json'
+title:   'Word Set'
+type:    object
+
+description: 'A *WordSet* is a set of related lexemes across languages. '
```

Comment:
```suggestion
description: 'A *WordSet* is a set of related lexemes across languages. This could be a cognate set, a loandword set, or some other crosslinguistic grouping.'
```